src/main/java/net/mcreator/ui/action/impl/AboutAction.java

```diff
@@ -49,8 +49,9 @@ public static void showDialog(Window parent) {
 				L10N.t("dialog.about.option.eula"), L10N.t("dialog.about.option.third_party_licenses"),
 				L10N.t("dialog.about.option.donate") };
 		int n = JOptionPane.showOptionDialog(parent,
-				L10N.t("dialog.about.message", Launcher.version.major, Launcher.version.getFullString(),
-						MCreatorApplication.WEB_API.getUpdateInfo().getLatestMajor(), GeneratableElement.formatVersion,
+				L10N.t("dialog.about.message", Launcher.version.major, Launcher.version.getFullString(), MCreatorApplication.isInternet ?
+						MCreatorApplication.WEB_API.getUpdateInfo().getLatestMajor() :
+						L10N.t("dialog.about.latest_version_not_applicable"), GeneratableElement.formatVersion,

```

Comment:
You are calling key that does not exist here

Reply:
:heavy_check_mark: Fixed